### PR TITLE
feat/864: enable bargeIn when minBargeinWordCount is 0

### PR DESCRIPTION
**Pull request by rammohan-y**
enable bargeIn configuration when minBargeinWordCount is 0

https://github.com/jambonz/jambonz-feature-server/issues/864

### CODE REVIEW

File: lib/tasks/config.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ class TaskConfig extends Task {
         'finishOnKey', 'input', 'numDigits', 'minDigits', 'maxDigits',
         'interDigitTimeout', 'bargein', 'dtmfBargein', 'minBargeinWordCount', 'actionHook'
       ].forEach((k) => {
-        if (this.bargeIn[k]) this.gatherOpts[k] = this.bargeIn[k];
+        if (this.bargeIn[k] !== undefined) this.gatherOpts[k] = this.bargeIn[k];
```
Comment (xquanluu) [inline lib/tasks/config.js:37]: I think
const val = this.bargeIn[k];
if (val !== undefined && val !== null) this.gatherOpts[k] = val;

what do you think?

**Comment by xquanluu** [comment]
I'm good with this.

File: lib/tasks/config.js (hunk 1 above)
Comment (rammohan-y) [reply lib/tasks/config.js]: @xquanluu made changes please review

**Comment by rammohan-y** [comment]
@davehorton can you please help with approval and merge of this PR

**Comment by vdharashive** [comment]
@xquanluu we have tested it with config bargin by setting minwordcount and setting different mode and it's working as expected. Just the way it is done for gather 

@davehorton pls approve

**Comment by xquanluu** [comment]
Look good to me.

**Review by davehorton — APPROVED** [review]

**Pull request merged**